api/PeriodicSyncEvent.json

```
@@ -0,0 +1,146 @@
+{
+  "api": {
+    "PeriodicSyncEvent": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": "80"
+          },
+          "chrome_android": {
+            "version_added": "80"
+          },
+          "edge": {
+            "version_added": "80"
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "67"
+          },
+          "opera_android": {
+            "version_added": "57"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": false
+          },
+          "webview_android": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
```

The spec itself says

> It is not a W3C Standard nor is it on the W3C Standards Track

So I'd like to take the spec at its word here (and throughout the PR)

```suggestion
          "standard_track": false,
```